test/torch/tensors/test_chinese_remainder.py

```diff
@@ -0,0 +1,97 @@
+import torch
+import syft
+
+# from syft.frameworks.torch.tensors.interpreters import syft.CRTTesor
+# from syft import syft.CRTTesor
+
+
+def test_wrap():
+    """
+    Test the .on() wrap functionality for syft.CRTTesor
+    """
+
+    res_3 = torch.tensor([[1, 2], [0, 1]]).fix_prec(field=3, precision_fractional=0)
+    res_7 = torch.tensor([[3, 4], [5, 6]]).fix_prec(field=7, precision_fractional=0)
+    residues = {3: res_3, 7: res_7}
+
+    x = syft.CRTTensor().on(residues, wrap=False).wrap()  # TODO see why I have to do that
+
+    assert isinstance(x, torch.Tensor)
+    assert isinstance(x.child, syft.CRTTensor)
+    assert isinstance(x.child.child, dict)
+
+
+def test__str__():
+    res_3 = torch.tensor([[1, 2], [0, 1]]).fix_prec(field=3, precision_fractional=0)
+    res_7 = torch.tensor([[3, 4], [5, 6]]).fix_prec(field=7, precision_fractional=0)
+    residues = {3: res_3, 7: res_7}
+
+    crt = syft.CRTTensor(residues).wrap()
+
+    assert isinstance(crt.__str__(), str)
+
+
+def test_eq():
+    res_3a = torch.tensor([1, 2]).fix_prec(field=3, precision_fractional=0)
+    res_7a = torch.tensor([3, 4]).fix_prec(field=7, precision_fractional=0)
+    residues_a = {3: res_3a, 7: res_7a}
+    crt_a = syft.CRTTensor(residues_a).wrap()
+
+    crt_b = syft.CRTTensor(residues_a).wrap()
+
+    res_3c = torch.tensor([1, 2]).fix_prec(field=3, precision_fractional=0)
+    res_7c = torch.tensor([4, 4]).fix_prec(field=7, precision_fractional=0)
+    residues_c = {3: res_3c, 7: res_7c}
+    crt_c = syft.CRTTensor(residues_c).wrap()
+
+    assert crt_a == crt_b
+    assert not (crt_a == crt_c)
+
+
+"""
+def test_ne():
```

Comment:
Why are these commented out?

Reply:
It's commented out because the __ne__ method is not hooked for any other syft tensor, and also because I think some upstream work (in native.py or something like that) needs to be done to enable the use of `!=`.
Maybe hooking `__ne__` for all syft tensors can be another PR?

Reply:
Definitely